### ISSUE TITLE
Gauntlet check required?

### ISSUE THREAD

**Issue by flohgh** [tewtal/itemrandomizerweb#26]
Hi,

First of all, thanks for the great work put in this randomizer, I have lots of fun watching races and playing it.

In your 2.5 version changelog, you wrote "Wall jumping up to early gauntlet without hi-jump will no longer be enforced for completion. (Note that going there early if you can is still going to be a good choice most of the time)". Is that still true? Because if it is I got a seed that did not enforce this (TX5756082).

I went everywhere I could without speed or high jump, except gauntlet (I did not know how to get to it before this) :

- crateria (minus gauntlet)
- all of brinstar but waterway
- all upper norfair
- only 2 items in wrecked ship : right super+gravity. Not the Etank since I didn't have high jump or speed to go through the first room with water (before spiky death room).

So the only place I could theoretically go to was gauntlet. And I did, after half an hour of learning the not so easy walljumps to get there (of course I did not have bombs or space jump either). And there was high jump, that unlocked litterally everything : 

- lower norfair (lava dive)
- WS etank (where gravity was)
- suitless outer maridia (which I did, since I forgot about WS Etank :()

So, long story short, is that a bug or did you change that logic at some point?

**Comment by flohgh** [comment]
Well, reading the code, turns out I could have used spring ball jump to reach wrecked ship etank, so it was not locked away by hi jump... So, this is not a bug ;)
